test/e2e/suites/base/schedule_multi_template_test.go

```diff
@@ -234,4 +239,168 @@ var _ = ginkgo.Describe("[ScheduleMultiTemplate] schedule multi template resourc
 			})
 		})
 	})
+
+	ginkgo.Context("VolcanoJob scheduling", func() {
+		var (
+			volcanoJobCRD       apiextensionsv1.CustomResourceDefinition
+			volcanoJobNamespace string
+			volcanoJobName      string
+			volcanoJobObj       *unstructured.Unstructured
+			volcanoJobGVR       schema.GroupVersionResource
+		)
+
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("create VolcanoJob CRD on karmada control plane", func() {
+				err := yaml.Unmarshal([]byte(volcanoJobCRDYAML), &volcanoJobCRD)
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+				framework.CreateCRD(dynamicClient, &volcanoJobCRD)
+				ginkgo.DeferCleanup(func() {
+					framework.RemoveCRD(dynamicClient, volcanoJobCRD.Name)
+				})
+			})
+		})
+
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("propagate VolcanoJob CRD to all clusters", func() {
+				cpp := testhelper.NewClusterPropagationPolicy("volcano-job-cpp", []policyv1alpha1.ResourceSelector{
+					{
+						APIVersion: volcanoJobCRD.APIVersion,
+						Kind:       volcanoJobCRD.Kind,
+						Name:       volcanoJobCRD.Name,
+					},
+				}, policyv1alpha1.Placement{
+					ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+						ClusterNames: framework.ClusterNames(),
+					},
+				})
+
+				framework.CreateClusterPropagationPolicy(karmadaClient, cpp)
+				framework.WaitCRDPresentOnClusters(karmadaClient, framework.ClusterNames(),
+					fmt.Sprintf("%s/%s", volcanoJobCRD.Spec.Group, "v1alpha1"), volcanoJobCRD.Spec.Names.Kind)
+				ginkgo.DeferCleanup(func() {
+					framework.RemoveClusterPropagationPolicy(karmadaClient, cpp.Name)
+				})
+			})
+		})
+
+		ginkgo.It("base case: propagate VolcanoJob to one cluster", func() {
+			volcanoJobNamespace = testNamespace
+			volcanoJobName = fmt.Sprintf("volcanojob-%s", rand.String(RandomStrLength))
+
+			ginkgo.By("create VolcanoJob on karmada control plane", func() {
+				volcanoJobObj = &unstructured.Unstructured{}
+				err := yaml.Unmarshal([]byte(volcanoJobCRYAML), volcanoJobObj)
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+				volcanoJobObj.SetNamespace(volcanoJobNamespace)
+				volcanoJobObj.SetName(volcanoJobName)
+
+				volcanoJobGVR = schema.GroupVersionResource{
+					Group:    volcanoJobObj.GroupVersionKind().Group,
+					Version:  volcanoJobObj.GroupVersionKind().Version,
+					Resource: "jobs",
+				}
+
+				_, err = dynamicClient.Resource(volcanoJobGVR).Namespace(volcanoJobNamespace).
+					Create(context.Background(), volcanoJobObj, metav1.CreateOptions{})
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+				ginkgo.DeferCleanup(func() {
+					err := dynamicClient.Resource(volcanoJobGVR).Namespace(volcanoJobNamespace).
+						Delete(context.Background(), volcanoJobName, metav1.DeleteOptions{})
+					if err != nil && !apierrors.IsNotFound(err) {
+						// Ignore not found errors during cleanup, but other errors should be handled
```

Comment:
I asked the same question at https://github.com/karmada-io/karmada/pull/6934#discussion_r2544257410. Why could the error be `Not Found`?

Reply:
To be honest, this part was auto-completed by the code. For the current test case, it indeed does not encounter a "not found" situation.
To avoid any misunderstanding, let me delete it.